github.go

```diff
@@ -22,6 +22,7 @@ import (
 	"log"
```

Comment:
Perhaps we should escape at the time of applying template, i.e. check if it is html or markdown output expectation and then escape accordingly.